pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/impl/SegmentDictionaryCreator.java

```diff
@@ -345,6 +345,16 @@ public int[] indexOfMV(Object value) {
           indexes[i] = _stringValueToIndexMap.getInt(multiValues[i]);
         }
         break;
+      case BIG_DECIMAL:
```

Comment:
We don't have MV big-decimal, so let's not add it for now. We can add it when adding the MV support for it

Reply:
yeah i was confused too on this.